fem/transfer.hpp

```diff
@@ -40,6 +41,11 @@ class GridTransfer
    OperatorHandle fw_t_oper; ///< Forward true-dof operator
    OperatorHandle bw_t_oper; ///< Backward true-dof operator
 
+   bool use_device;
+   bool verify_solution;
```

Comment:
I'm not sure if we want to keep this `verify_solution` around. It looks useful for debugging/development of the transfer classes, but not like something the user will need to use

Reply:
@v-dobrev, @helloworld922 any thoughts? It was useful while we developed the code.

Reply:
I agree with Will, perhaps we could add the verification code as helper functions for unit testing purposes but I can't think of a reason a user would find this feature useful.

Reply:
okay, I'll remove it and reduce the duplicate mult/multTranspose methods